bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -758,11 +758,26 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 >hgrid=0.9x1.25 maxpft=79 mask=gx1v7 use_cn=.true. use_crop=.true. irrigate=.true. glc_nec=10 do_transient_pfts=.false.
 </init_interp_attributes>
 
+<init_interp_attributes sim_year="2000" use_cndv=".false." use_fates=".false." lnd_tuning_mode="clm5_0_cam6.0"
```

Comment:
@adamrher note I added these changes here for init_interp_attributes, otherwise it matches to the ne120 file which we don't really want for low resolutions grids. Let me know if you have questions, comments, or concerns.

Reply:
got it. makes sense.